source/chapters/controlling_mixxx.rst

```diff
@@ -56,7 +56,7 @@ Using a Keyboard
 
    Mixxx Keyboard shortcuts (for en-us keyboard layout)
 
-   :download:`Download the image <../_static/Mixxx-111-Keyboard-Mapping.png>`
+   :download:`Download the image <../_static/Mixxx-200-Keyboard-Mapping.png>`
```

Comment:
220?

Reply:
fixed.
I missed the filename of the actal figure a few lines above. also fixed